src/common/components/ReviewItem/ProfileSchool.tsx

```diff
@@ -1,15 +1,32 @@
 import { reviewItemTheme, type ReviewItemVariants } from "./ReviewItem.theme";
+import { SchoolIcon } from "@/common/components/CustomIcon";
 
 export type ProfileSchoolProps = ReviewItemVariants & {
   courseCode: string;
+  reviewedUniversityId: number;
 };
 
-export const ProfileSchool = ({ courseCode }: ProfileSchoolProps) => {
+export const ProfileSchool = ({
+  courseCode,
+  reviewedUniversityId,
```

Comment:
thought: it would probably be easier to pass primsa's `UniversityAbbreviation` like `'SMU' | 'NTU' | 'NUS'` instead of the id since `<SchoolIcon/>` takes in the abbrv instead of id

note: think of ids as a db-level concept which should never be overflown into application logic

Reply:
oversight on my end, fixed! https://github.com/AfterClass-io/afterclass.io-v2/pull/104/commits/88380b386911aa02b4bac43d0a49fc8f760732be